source/reference/array.rst

```diff
@@ -236,7 +236,193 @@ Unevaluated Items
 
 |draft2019-09|
 
-Documentation Coming Soon
+The ``unevaluatedItems`` keyword applies to any values not evaluated
+by an ``items``, ``prefixItems``, or ``contains`` keyword. Just as
+``unevaluatedProperties`` affects only **properties** in an object,
+``unevaluatedItems`` affects only **item**-related keywords.
+
+For this first example, let's assume you want to allow lists of items
+that begin with either ``1`` or ``"A"``, but anything after must be ``2``.
+
+.. schema_example::
+
+    {
+      "oneOf": [{"prefixItems": [{"const": 1}]}, {"prefixItems": [{"const": "a"}]}],
+      "items": {"const": 2}
+    }
```

Comment:
Hmm, do you think it's important to mention somewhere that (line 257) ``items`` doesn't "see inside" any instances of ``oneOf``, ``anyOf``, or ``allOf`` in the same subschema? I mention in line 323 that ``unevaluatedItems`` doesn't "see inside" cousins, but it's difficult to weave those two threads together. Otherwise, I may scrap this entire example because you're right, it's pretty much the same as extending the tuple below.

Reply:
Yes, I think it's important to mention because that's the reason `unevaluatedItems` exists. Going with my suggestion of starting with a simple open tuple and showing various ways of referencing it to extend or close it, you could start by showing how trying to extend or close  the referenced tuple with `items` doesn't work. Then you can introduce `unevaluatedItems` as the solution to that problem.